calico/common.py

```diff
@@ -282,6 +282,12 @@ def complete_logging(logfile=None,
             file_handler.setFormatter(formatter)
             root_logger.addHandler(file_handler)
 
+    # Optimization: disable all logging below the minimum level that we care
+    # about.  The global "disable" setting is the first thing that gets checked
+    # in the logging framework so it's the fastest way to disable logging.
+    min_log_level = min(file_level, syslog_level, stream_level)
+    logging.disable(min_log_level - 1)
+
```

Comment:
Actually, I think that' s a good spot!  "none" is a valid value to use in the config
```
LOGLEVELS = {"none":      None,
             "debug":     logging.DEBUG,
             "info":      logging.INFO,
             "warn":      logging.WARNING,
             "warning":   logging.WARNING,
             "err":       logging.ERROR,
             "error":     logging.ERROR,
             "crit":      logging.CRITICAL,
             "critical":  logging.CRITICAL}
```